apps/fishing-map-e2e/src/support/map/eez-download.po.ts

```diff
@@ -71,7 +77,8 @@ export const testCsvOptions = (
 ) => {
   cy.log(description)
   deleteDownloadsFolder()
-  buttons.forEach((button) => cy.getBySel(container).findBySel(button).click())
+  // We need force because the buttons are covered by the active element
+  buttons.forEach((button) => cy.getBySel(container).findBySel(button).click({ force: true }))
```

Comment:
If this is used in other places I'd extract it to a custom command as suggested [here](https://stackoverflow.com/a/61325254)
```node
Cypress.Commands.add('forceClick', {prevSubject: 'element'}, (subject, options) => {
  cy.wrap(subject).click({force: true})
});
```
What do you think?